js/modal.js

```diff
@@ -0,0 +1,134 @@
+const projectWrapper = document.querySelector('#works-wrapper');
+const modal = document.querySelector('#modal');
+const closeModal = document.querySelector('#close-modal');
+const modalContent = document.querySelector('#modal-content');
+closeModal.addEventListener('click', () => modal.classList.add('hidden'));
+
+const projects = [
+  {
+    id: 1,
+    name: 'Tonic',
+    company: 'Canopy',
+    position: 'Back End Dev',
+    year: '2022',
+    shortDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'tonic1.png',
+    technologies: ['html', 'css', 'javaScript'],
+    liveVersion: 'https://eugeneiyukhun.github.io/My-Portfolio-Desktop-Version/',
+    source: 'https://github.com/Eugeneiyukhun/My-Portfolio-Desktop-Version',
+  },
+  {
+    id: 2,
+    name: 'Multi-Post Stories',
+    company: 'Canopy',
+    position: 'Back End Dev',
+    year: '2022',
```

Comment:
- [ ] You should stick to the design as much as possible. Please fix the spacing between `hx` tag and list items. please see image below for reference.
### Your design
![your](https://user-images.githubusercontent.com/81727988/210963259-da099703-0ad3-48ac-ba1d-b10662de405b.png)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/210963284-4fabcbba-02d9-4f7b-b8fd-37a9497ec5f6.png)